packages/arb-ts/integration_test/arb-bridge.test.ts

```diff
@@ -352,18 +330,20 @@ describe('CustomToken', () => {
       )
       l2CustomToken = await customL2TokenFactory.deploy(
         bridge.arbTokenBridge.address,
-        l1CustomToken.address
+        l1CustomToken.address,
+        { gasLimit: 100000000 }
```

Comment:
Should we be hardcoding gas values like this? If we're getting a wrong `eth_estimateGas` we should probably look into it

Reply:
yes, this is a lazy patch atm